ext/systemd/pe-bolt-server.service

```diff
@@ -6,9 +6,10 @@ After=syslog.target network.target
 Type=simple
 EnvironmentFile=-/etc/sysconfig/pe-bolt-server-service
 EnvironmentFile=-/etc/default/pe-bolt-server-service
-ExecStart=/opt/puppetlabs/puppet/bin/pe-bolt-server $PE_BOLT_SERVER_OPTIONS --foreground
+Environment=GEM_HOME=/opt/puppetlabs/server/apps/bolt-server/lib/ruby/gems/2.4.0
+ExecStart=/opt/puppetlabs/server/apps/bolt-server/bin/bolt-server $PE_BOLT_SERVER_OPTIONS --foreground
```

Comment:
Do we want to change the var to BOLT_SERVER_OPTIONS? Minor, but also less confusing. I don't feel strongly either way.

Reply:
I'm tempted to change 'pe bolt server' to 'bolt server' everywhere, but as long as the service is named pe-bolt-server I think the environment variable should stay that way too.